wdl/tasks/Utility/MergeVCFs.wdl

```diff
@@ -0,0 +1,159 @@
+version 1.0
+
+import "../../structs/Structs.wdl"
+
+workflow MergeVCFs {
```

Comment:
Is this taken from the GATK-SV pipeline?

And is this integrated into other pipelines in this repo? 
I checked out your branch and deleted that file, then went to validate all pipelines. All WDLs validated, so I suspect it is not integrated.

If it is indeed not integrated into other pipelines here, do you mind making a separate PR?

Reply:
This was something I wrote for the GMKF data import.  It doesn't belong in this PR.  I'll remove it.